Source/WebCore/css/CSSCounterStyleDescriptors.cpp

```diff
@@ -67,6 +67,28 @@ static String symbolToString(const CSSValue* value)
     return primitiveValue.stringValue();
 }
 
+static CSSCounterStyleDescriptors::AdditiveSymbols translateAdditiveSymbolsFromStyleProperties(const StyleProperties& properties)
+{
+    auto additiveSymbols = properties.getPropertyCSSValue(CSSPropertyAdditiveSymbols).get();
+    if (!is<CSSValueList>(additiveSymbols))
+        return { };
+    auto& list = downcast<CSSValueList>(*additiveSymbols);
+    CSSCounterStyleDescriptors::AdditiveSymbols result;
+    for (auto& additiveValue : list) {
+        if (auto* additiveTuple = dynamicDowncast<CSSValueList>(&additiveValue)) {
```

Comment:
We should really be using CSSValuePair here: https://searchfox.org/wubkat/rev/98761d6864f7dac2f22655a51bb8a00a2172edc3/Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp#8227

like consumeCounterStyleRange does